tsconfig.json

```diff
@@ -1,9 +1,9 @@
 {
     "compilerOptions": {
-        "module": "CommonJS",
```

Comment:
@garronej this should not have been removed,

Reply:
Yes, my bad.  
The default config is for CommonJS and we overwrite some values when  building for ESM....